.github/workflows/release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
-      - run: npm install --force
+      - run: npm install
```

Comment:
Following recommendation from this PR
https://github.com/cloudscape-design/actions/pull/27